src/Wallabag/CoreBundle/Form/Type/UserInformationType.php

```diff
@@ -10,8 +10,8 @@ class UserInformationType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name', 'text')
-            ->add('email', 'text')
+            ->add('name', 'text', array('required' => true))
+            ->add('email', 'email')
```

Comment:
> The required isn't needed because, since the form is mapped to the User entity, it will apply annotation from the entity to the form.

It's just more pratical to use with mobile keyboards when entering your email (@ shows up). Nothing to do with server checking. Right ?